guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,600 @@
+# Deploying clustered Akka applications on Kubernetes
+
+## The challenge
+
+Deploying an Akka cluster on Kubernetes presents the following challenges:
+
+* One or more seed notes must be specified to bootstrap the cluster.
+* Kubernetes requires apps to be containerized before it will run them.
+* Kubernetes requires configuration for stateful applications.
+
+We will show how to containerize the application using [SBT](http://www.scala-sbt.org/) or [Maven](https://maven.apache.org/) and how to configure Kubernetes resources, particularly [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) to establish Akka cluster for your application.
```

Comment:
It's actually SBT and Maven that's doing the containerization, so the sentence above is correct. The result of the build process is a Docker image.